docs/guides/es2k/running-infrap4d.md

```diff
@@ -0,0 +1,282 @@
+# Run Infrap4d on Intel IPU E2100
+
+- [1. Overview](#1-overview)
```

Comment:
This document is a list of eight steps. Don't make it unnecessarily complicated.
- Omit the **Overview** heading. Just have a couple of short sentences between the document title and the first numbered section. The reader isn't stupid. They can figure out that this is an overview.
- Make each of the eight steps a level-1 heading.

In other words:
# Running Infrap4d on Intel IPU E2100
(intro)
## 1. Extract libraries
## 2. Set up environment
## 3. Install TLS certificates
## 4. Generate pipeline binary
## 5. Start infrap4d
## 6. Configure pipeline
## 7. Configure forwarding rule
## 8. Send test traffic

Reply:
I made a section for running infrap4d on ACC since we will have documentation for Host in the future and it will be part of the same document. So, something like:
Running Infrap4d on Intel IPU E2100
(intro)
1. Running Infrap4d on ACC
   - 1.1 Extract libraries
   - 1.2 Set up environment
   - 1.3.. etc
3. Running Infrap4d on Host
 - 3.1 Extract RPMs
 - 3.2 Setup environment
 - 3.3 ... etc

If we plan to modify the structure again when we add documentation for host or maintain a separate document for instructions to run infrap4d on host, I can change it to the proposed format.

Reply:
Changed to proposed format. Most of the headings are level 1 and level 2, hence changed to Title case